akka-http-core/src/main/scala/akka/http/impl/engine/http2/client/PersistentConnection.scala

```diff
@@ -84,7 +94,8 @@ private[http2] object PersistentConnection {
         connected:    Future[OutgoingConnection],
         requestOut:   SubSourceOutlet[HttpRequest],
         responseIn:   SubSinkInlet[HttpResponse],
-        connectsLeft: Option[Int]
+        connectsLeft: Option[Int],
+        embargo:      FiniteDuration
```

Comment:
I find the naming slightly confusing, maybe `embargo` -> `lastEmbargo`, `nextEmbargo` -> `embargo`?